### PR TITLE
pkg/controller: fix staticcheck warning

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:

fix staticcheck warning

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by mortent — APPROVED** [review]
/lgtm

File: pkg/controller/controller_utils.go [hunk 1]
```diff
@@ -508,10 +508,10 @@ func validateControllerRef(controllerRef *metav1.OwnerReference) error {
 	if len(controllerRef.Kind) == 0 {
 		return fmt.Errorf("controllerRef has empty Kind")
 	}
-	if controllerRef.Controller == nil || *controllerRef.Controller != true {
```
Comment (deads2k) [inline pkg/controller/controller_utils.go]: I don't care for `!*foo`.  Can you turn off that linting rule?

Comment (hwdef) [reply pkg/controller/controller_utils.go]: Should we follow the rules of static checking？

**Pull request closed without merging**